app/cinema/hall.py

```diff
@@ -0,0 +1,11 @@
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie_name: str, customers: str,
+                      cleaning_staff: str) -> None:

```

Comment:
Invalid type hint for `customers` nad `cleaning_staff`
```suggestion
    def movie_session(self, movie_name: str, customers: list[Customer],
                      cleaning_staff: Cleaner) -> None:

```